.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Create Release
```

Comment:
I think for now we want to not do this, since the publish process is a bit complicated right now since we have many sub dependencies and things that depend on other crates being released before others.